packages/account/src/Components/api-token/api-token.tsx

```diff
@@ -231,45 +216,35 @@ const ApiToken = ({ footer_ref, is_app_settings, overlay_ref, setIsOverlayShown
                                                 <div className='da-api-token__checkbox-wrapper'>
                                                     <ApiTokenCard
                                                         name='read'
-                                                        value={values.read}
-                                                        setFieldValue={setFieldValue}
                                                         display_name={localize('Read')}
                                                         description={localize(
```

Comment:
can we make this to Localize 
This description is rendered as children of Text component

Reply:
Should I do it for description only or `InlineNoteWithIcon` component as well?